canonical-websites/www.canonical.com
# Update to latest version of run script

## Done

* Update to latest verion of the run script

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/](http://0.0.0.0:8002/)
4. Make sure the website still works
